syz-manager/manager.go

```diff
@@ -822,7 +827,8 @@ func (mgr *Manager) runInstanceInner(index int, instanceName string, injectLog <
 	}
 	cmd := instance.FuzzerCmd(args)
 	_, rep, err := inst.Run(mgr.cfg.Timeouts.VMRunningTime, mgr.reporter, cmd,
-		vm.ExitTimeout, vm.StopChan(mgr.vmStop), vm.InjectOutput(injectLog))
+		vm.ExitTimeout, vm.StopChan(mgr.vmStop), vm.InjectOutput(injectLog),
+		vm.EarlyFinishCb(earlyFinishCb))
```

Comment:
Let's create the callback right in this function, it looks like creating it in runInstance only adds code/arguments.